starters/deno-oak-denodb/src/main.ts

```diff
@@ -1,11 +1,27 @@
-import { Application, Context, oakCors, Router } from '../deps.ts';
+import { Application, Context, oakCors, Router, log, config } from '../deps.ts';
 import { corsAllowedOrigins } from './util/cors_allowed_origins.ts';
 import { API_HOST, PORT } from './config/environment.ts';
 
 const app = new Application();
 const port = +PORT || 3333;
-
+const { LOG_LEVEL } = config({ safe: true });
```

Comment:
This can be exported from `./config/environment.ts` and added to the import in line 3